src/pages/iframes/comments/[dtpId].tsx

```diff
@@ -22,15 +23,17 @@ const CommentsIframePage: NextPage<CommentsIframePageProps> = ({
   }
 
   return (
-    <CommentsProvider initComments={comments}>
-      <CommentList />
+    <UserProvider>
```

Comment:
We can probably move this into `_app.tsx`

Reply:
Done